libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -221,6 +230,21 @@ void SRV_Channels::set_output_pwm_chan(uint8_t chan, uint16_t value)
     }
 }
 
+// set output value for a specific function channel as a pwm value with loop based timeout
```

Comment:
comment out of date

Reply:
I've expanded to 3 lines of comments now. Should I make that a multi line block comment?

Reply:
doesn't matter to me

Reply:
Change implemented.